apidocs/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../.eslintrc",
+  "rules": {
+    "quotes": ["off"]
```

Comment:
@bajtos PATL, this will end up ignoring double quotes in both `describe-builtin-models.js` AND `loopback-core.js`
do you think it's fine?